packages/components/src/font-size-picker/index.js

```diff
@@ -1,22 +1,34 @@
+
 /**
- * External dependencies
+ * WordPress dependencies
```

Comment:
Looks like there are two WordPress dependency comments, would be great to merge the imports into one section.

Reply:
🤦‍♂ how ever did that happen?